spec/support/helpers/session_helpers.rb

```diff
@@ -2,17 +2,17 @@ module Features
   module SessionHelpers
     def sign_up_with(email, password, confirmation)
       visit new_user_registration_path
-      fill_in 'Email', with: email
-      fill_in 'Password', with: password
-      fill_in 'Password confirmation', :with => confirmation
+      fill_in 'user[email]', with: email
+      fill_in 'user[password]', with: password
+      fill_in 'user[password_confirmation]', :with => confirmation
       click_button 'Sign up'
     end
 
     def signin(email, password)
       visit new_user_session_path
-      fill_in 'Email', with: email
-      fill_in 'Password', with: password
-      click_button 'Sign in'
+      fill_in 'user[email]', with: email
+      fill_in 'user[password]', with: password
+      click_button 'Log in'
```

Comment:
this helper was pulled from our solution branch - which currently passes all the tests. I'm not super familiar with devise either....